src/main/jbake/templates/index.ftl

```diff
@@ -5,7 +5,7 @@
     <section class="main-banner">
         <div class="container">
           <h1 class="fade-in-up one">
-            The Open Source
+            Open Source
```

Comment:
Makes sense. For me it was more a marketing statement, in sense "the famous one", or "you should have known about it".